README.md

```diff
@@ -130,6 +130,9 @@ To keep all those hundreds of source files from ending up in the same directory,
 
 First, create them as groups (little yellow "folders") within the group with your project's name in Xcode's Project Navigator. Then, for each of the groups, link them to an actual directory in your project path by opening their File Inspector on the right, hitting the little gray folder icon, and creating a new subfolder with the name of the group in your project directory.
 
+However, in case of a complex project, splitting the files based on the features is a good idea to keep in mind. 
```

Comment:
Maybe we should be more opinionated here, stating that once it makes sense to endow ones project tree with more depth one should cluster by feature, not by function, as in a well structured project separate features should be as loosely coupled as possible, while coupling between one component's feature is natural.

Reply:
Actually, jpmcglone already opened a well written issue about that.